### ISSUE TITLE
 queuednotification execution error

### ISSUE THREAD

**Issue by Riesuu** [glpi-project/glpi#9289]
Hello,
I don't know why but the service queuednotification started to crash
![image](https://user-images.githubusercontent.com/17006344/125783001-a213d93a-d791-474b-8c29-d241ddb6df90.png)
![image](https://user-images.githubusercontent.com/17006344/125783024-9ddde70c-aa5e-4bf5-8f58-da695e1901db.png)
[php-errors.log](https://github.com/glpi-project/glpi/files/6822825/php-errors.log)

GLPI 9.5.5

**Comment by cedric-anne** [comment]
Hi,

According to what I found in the code, this is related to the resizing of a specific image that should be sent in a notification.
Please try to find which image causes this issue and attach it here as it may help us to investigate.

Regards

**Comment by Riesuu** [comment]
Hi,
I find the mail, I delete it and I launch the service?

**Comment by cedric-anne** [comment]
The image is related to a document that have to be sent with queued notifications that can be found on `/front/queuednotification.php`.
So if you want to find the image that causes this issue, you have to inspect these notifications.

**Comment by Riesuu** [comment]
The problem is solved, as a ticket collected via the anonymous collector was a problem. This one had 80 follow-ups with many duplicates on the photos. The photos are only 2 to 200kb. So I deleted the ticket

![image](https://user-images.githubusercontent.com/17006344/125796525-32dc874f-e33f-457d-8687-c5e23069c7dd.png)
![image](https://user-images.githubusercontent.com/17006344/125796618-d15e10c1-4747-439d-863d-bccf1f26445c.png)


I solved the ticket on 13/07 at 15:22. It was probably not able to send the notifications.

Thanks for ur help
